packages/api/src/server/application/service/application.service.ts

```diff
@@ -88,7 +88,7 @@ export class ApplicationService {
         propertyIdentifier,
         env
       });
-      console.log(agendaResponse);
+      this.logger.log('Agenda', JSON.stringify(agendaResponse));
```

Comment:
Do we need the entire response?
We can add this response into different levels right?